department-of-veterans-affairs/abd-vro
# MCP-1919 Rename RabbitMq queues from diagnostic code to diagnosis

## What was the problem?
Rename the queues from 7101 → “hypertension” and 6602 → “asthma”, since we are adopting that naming convention for future conditions.

Associated tickets or Slack threads:

- [MCP-1919](https://amida.atlassian.net/browse/MCP-1919)

## How does this fix it?[^1]
Changes the queue names and adjusts tests to accept new queue names

## How to test this PR
- Start the app
- Navigate to http://localhost:15672/#/queues and confirm that the queue names contain the diagnoses and not their diagnostic codes


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
